runtime/src/bank.rs

```diff
@@ -6302,13 +6302,30 @@ impl Bank {
             .map(|epoch_stakes| epoch_stakes.total_stake())
     }
 
+    /// Get the total epoch stake for the current Bank::epoch
+    pub fn get_current_epoch_total_stake(&self) -> Option<u64> {
+        self.epoch_stakes
+            .get(&self.epoch.saturating_add(1))
```

Comment:
Can you use the `current_epoch_stakes` method here instead? (And also in `get_current_epoch_vote_accounts`)

Reply:
I can, I guess I was just wary of using the method that can panic.

Reply:
Understandable but this is safe because whenever banks are created they will always add an entry in epoch stakes at `self.epoch + 1`

Reply:
Added!